tests/requests/rds/db_snapshot_tests.rb

```diff
@@ -0,0 +1,73 @@
+Shindo.tests('Fog::Rds[:aws] | db snapshot requests', ['aws']) do
+
+  @snapshot_format = {
+    'AllocatedStorage'     => Integer,
+    'AvailabilityZone'     => Fog::Nullable::String,
+    'Engine'               => String,
+    'EngineVersion'        => String,
+    'InstanceCreateTime'   => Time,
+    'DBInstanceIdentifier' => String,
+    'Iops'                 => Fog::Nullable::Integer,
+    'MasterUsername'       => String,
+    'Port'                 => Fog::Nullable::Integer,
+    'Status'               => String,
+    'StorageType'          => String,
+    'SnapshotType'         => String
+  }
+
+  @snapshots_format = {
+    'requestId'   => String
+  }
+
+  @snapshot_copy_result = {
+    'requestId'   => String,
+    'snapshotId'  => String
+  }
+
+  Fog.mock!
```

Comment:
I don't think this should be here (it would be set or not at a higher level). The test you mention is broken on master I'm pretty sure, so we can consider this good even without this. If you can remove this I think we should be good to merge the rest. Thanks!